apps/site/src/pages/api/vote-application.api.ts

```diff
@@ -15,26 +15,24 @@ export default createBaseHandler<
   VoteApplicationRequestBody,
   VoteApplicationResponse
 >().put(async (req, res) => {
-  const member = await getMember({ email: req.body.email })
-    .then((mailchimpMember) => ({
-      id: mailchimpMember.id,
-      email: mailchimpMember.email_address,
-      merge_fields: mailchimpMember.merge_fields,
-    }))
-    .catch(() => {
-      return res.status(400).send({ error: true });
-    });
+  try {
+    const member = await getMember({ email: req.body.email }).then(
```

Comment:
It'd be great to avoid mixing `await` and `then`. Can we stick with `await` everywhere`? You can go for 

```suggestion
    const rawMember = await getMember({ email: req.body.email });
    const member = rawMember /* ... */
```